client/components/token-field/index.jsx

```diff
@@ -24,12 +25,27 @@ var TokenField = React.createClass( {
 	propTypes: {
 		suggestions: React.PropTypes.array,
 		maxSuggestions: React.PropTypes.number,
-		value: React.PropTypes.array,
 		displayTransform: React.PropTypes.func,
 		saveTransform: React.PropTypes.func,
 		onChange: React.PropTypes.func,
 		tokenStatus: React.PropTypes.func,
-		isBorderless: React.PropTypes.bool
+		isBorderless: React.PropTypes.bool,
+		value: function( props ) {
```

Comment:
How often are `propTypes` validators run? Are they run in production? Just curious if this would have a potential performance impact.